src/components/design-system/molecules/profile-header/profile-header.tsx

```diff
@@ -0,0 +1,65 @@
+import { Fragment } from 'react';
+import { Menu, Transition } from '@headlessui/react';
+import Image from 'next/image';
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ');
+}
+
+interface ProfileHeaderProps {
+  userImage: string;
+  menuItems: {
+    href: string;
+    label: string;
+    onClick?: (e: React.MouseEvent<HTMLElement>) => void;
+  }[];
+}
+
+export default function ProfileHeader({
+  userImage,
+  menuItems,
+}: ProfileHeaderProps) {
+  return (
+    <Menu as="div" className="relative ml-3">
+      <div>
+        <Menu.Button className="relative flex rounded-full bg-white text-sm focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2">
+          <span className="absolute -inset-1.5" />
+          <span className="sr-only">Open user menu</span>
+          <Image
+            className="h-8 w-8 rounded-full"
+            src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
```

Comment:
je crois qu'ici tu voulais mettre:
```suggestion
            src={userImage}
```